storage/redis/peer_store.go

```diff
@@ -8,48 +8,49 @@ import (
 
 	"github.com/chihaya/chihaya/bittorrent"
 	"github.com/chihaya/chihaya/storage"
-	"github.com/garyburd/redigo/redis"
+	redigo "github.com/garyburd/redigo/redis"
 )
 
-var namespacePrefix string
+var (
+	prefixedSeeder  string
+	prefixedLeecher string
+)
 
 type Config struct {
-	Namespace                 bool          `yaml:"namespace"`
-	Cntrl                     string        `yaml:"prefix"`
-	MaxNumWant                int           `yaml:"max_numwant"`
-	Host                      string        `yaml:"host"`
-	Port                      string        `yaml:"port"`
-	GarbageCollectionInterval time.Duration `yaml:"gc_interval"`
-	PeerLifetime              time.Duration `yaml:"peer_liftetime"`
+	Namespace    string        `yaml:"namespace"`
```

Comment:
can we call it as `KeyPrefix`? I feel it is more understandable than `Namespace`